Source/monster.h

```diff
@@ -31,6 +31,9 @@ struct Player;
 constexpr size_t MaxMonsters = 200;
 constexpr size_t MaxLvlMTypes = 24;
 
+constexpr int NightmareToHitBonus = 85;
+constexpr int HellToHitBonus = 120;

```

Comment:
These were previously defined in the anonymous namespace because they're not used outside of monster.cpp. It would make more sense to define `toHitSpecial()` in the cpp file rather than moving the variables to the header.

Reply:
Yes, it makes more sense. Moved `toHitSpecial` implementations to the .cpp file along with globals